scripts/exregional_run_analysis.sh

```diff
@@ -0,0 +1,594 @@
+# Make a list of the latest GFS EnKF ensemble
+#
+#-----------------------------------------------------------------------
+
+stampcycle=`date -d "${START_DATE}" +%s`
+minHourDiff=100
+loops="009"
+for loop in $loops; do
+  for timelist in `ls ${ENKF_FCST}/*.gdas.t*z.atmf${loop}s.mem080.nemsio`; do
+    availtimeyy=`basename ${timelist} | cut -c 1-2`
+    availtimeyyyy=20${availtimeyy}
+    availtimejjj=`basename ${timelist} | cut -c 3-5`
+    availtimemm=`date -d "${availtimeyyyy}0101 +$(( 10#${availtimejjj} - 1 )) days" +%m`
+    availtimedd=`date -d "${availtimeyyyy}0101 +$(( 10#${availtimejjj} - 1 )) days" +%d`
+    availtimehh=`basename ${timelist} | cut -c 6-7`
+    availtime=${availtimeyyyy}${availtimemm}${availtimedd}${availtimehh}
+    AVAIL_TIME=`echo "${availtime}" | sed 's/\([[:digit:]]\{2\}\)$/ \1/'`
+    AVAIL_TIME=`date -d "${AVAIL_TIME}"`
+
+    stamp_avail=`date -d "${AVAIL_TIME} ${loop} hours" +%s`
+
+    hourDiff=`echo "($stampcycle - $stamp_avail) / (60 * 60 )" | bc`;
+    if [[ ${stampcycle} -lt ${stamp_avail} ]]; then
+       hourDiff=`echo "($stamp_avail - $stampcycle) / (60 * 60 )" | bc`;
+    fi
+
+    if [[ ${hourDiff} -lt ${minHourDiff} ]]; then
+       minHourDiff=${hourDiff}
+       enkfcstname=${availtimeyy}${availtimejjj}${availtimehh}00.gdas.t${availtimehh}z.atmf${loop}s
+    fi
+  done
+done
+
+size=`du --apparent-size --block-size=1 --dereference ${ENKF_FCST}/${enkfcstname}.mem001.nemsio | grep -o '..........' | sed -n '1 p'`
+ls ${ENKF_FCST}/${enkfcstname}.mem001.nemsio > filelist03
+
+n=2
+while [[ $n -le 80 ]] ; do
+  
+  if [ $n -lt 10  ]; then
+    nn=00$n
+  else
+    nn=0$n
+  fi
+
+  size2=`du --apparent-size --block-size=1 --dereference ${ENKF_FCST}/${enkfcstname}.mem${nn}.nemsio | grep -o '..........' | sed -n '1 p'`
+
+  if [[ $size2 -lt $size ]]; then
+    print_info_msg "$VERBOSE"  "Bad GDAS member number ${nn}."
+  else
+    ls ${ENKF_FCST}/${enkfcstname}.mem${nn}.nemsio >> filelist03
+  fi
+
+  n=$((n + 1))
+
+done
+
+
+#
+#-----------------------------------------------------------------------
+#
+# set default values for namelist
+#
+#-----------------------------------------------------------------------
+
+cloudanalysistype=0
+ifsatbufr=.false.
+ifsoilnudge=.false.
+beta1_inv=1.0
+ifhyb=.false.
+
+# Determine if hybrid option is available
+memname='atmf009'
+nummem=`more filelist03 | wc -l`
+nummem=$((nummem - 3 ))
+if [[ ${nummem} -eq 80 ]]; then
+  print_info_msg "$VERBOSE" "Do hybrid with ${memname}"
+  beta1_inv=0.15
+  ifhyb=.true.
+  print_info_msg "$VERBOSE" " Cycle ${YYYYMMDDHH}: GSI hybrid uses ${memname} with n_ens=${nummem}" 
+fi
+
+
+#
+#-----------------------------------------------------------------------
+#
+# link or copy background files
+#
+#-----------------------------------------------------------------------
+
+FV3SARPATH=${CYCLE_DIR}
+cp_vrfy ${fixdir}/fv3_akbk                               fv3_akbk
+cp_vrfy ${fixdir}/fv3_grid_spec                          fv3_grid_spec
```

Comment:
These aren't really background files.

Reply:
I usually think everything on grid is for background but I added "grib configuration" for more accuracy description.

Reply:
Should this be grib or grid?

Reply:
It is grid.